recipes/fairlearn/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.4.2" %}
+
+package:
+  name: fairlearn
+  version: {{ version }}
+
+source:
+  url: https://github.com/fairlearn/fairlearn/archive/v{{ version }}.tar.gz
+  sha256: f602f572aa1aee36709d9edb38f5ff77c606a64caa92a8219328f6a127ec5411
+
+build:
+  number: 0
+
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
```

Comment:
You should remove these comments

Reply:
I had them there for us to remember what they are ;) removed now.